src/Spectre.libPeakFinder/DataTypes.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I'm pretty sure I've seen this file in #33.

Reply:
Yeah, it comes form the same commit. We have also considered moving the DataTypes.h to some common folder, since so many projects make use of it. It'd also make it easier to test the pipeline for the float types, since it'd render just a single file to be changed.

Reply:
I think this could be resolved in near future to go for extraction of this file. Not necessarily in this PR.